src/Microsoft.DotNet.Build.Tasks/AddDependenciesToProjectJson.cs

```diff
@@ -458,7 +463,7 @@ public NuGetVersion Version
 
         public string GetVersionString()
         {
-            return string.Join(".", _version.Major, _version.Minor, _version.Patch);
```

Comment:
Do you no longer require a 3-part version string?  older versions of NuGet used a 4 part version string.